sky/clouds/service_catalog/constants.py

```diff
@@ -1,6 +1,6 @@
 """Constants used for service catalog."""
 import os
 
-HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/skypilot-org/skypilot-catalog/master/catalogs'  # pylint: disable=line-too-long
+HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/romilbhardwaj/skypilot-catalog/k8s_images/catalogs'  # TODO - REVERT AFTER CATALOG PR IS MERGED. pylint: disable=line-too-long
```

Comment:
```suggestion
HOSTED_CATALOG_DIR_URL = 'https://raw.githubusercontent.com/skypilot-org/skypilot-catalog/master/catalogs'  # pylint: disable=line-too-long
```
reminder: Need this to be reverted if the catalog PR gets merged before this PR